test/README.txt.txt

```diff
@@ -0,0 +1,14 @@
+To run the unit tests:
+
+$ cd ${project.dir}
+$ ant -lib lib/saxon-he-10.6.jar   unit_tests
```

Comment:
1. the `readme file` name is Readme.md, please rename. 
2. the readme file shall be ONLY ONE and in the project root.

Reply:
We need a Makefile that 
1. install the necessary libraries
2. runs tests